database/migrations/2020_05_21_094708_remove_ordered_printed_from_voucher_states_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class RemoveOrderedPrintedFromVoucherStatesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('voucher_states', function (Blueprint $table) {
+            // I guess nice to have a record of how many rows were deleted?
+            $number_to_delete = App\VoucherState::whereIn('to', ['ordered', 'printed'])->count();
+
+            App\VoucherState::whereIn('to', ['ordered', 'printed'])->delete();
+            Log::info($number_to_delete . ' printed and ordered voucher_states records deleted on migration.');
```

Comment:
You want to leave this `Log::info()` in?

Reply:
The log is for historical info/ record. We do it in loads of places so we can easily look into activity in production. In particular around destructive or suspicious actions.

If you search the code base for `Log::info(` you'll see the types of things we log.

Reply:
It doesn't. I think if you aren't in App Laravel prefers the Log import not to be there.

You get that error when importing to tests.

Reply:
The 'has no effect' error. I have run this migration and rolled back multiple times on my local to check stuff.